save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPlugin.kt

```diff
@@ -65,6 +65,17 @@ class WarnPlugin(
         }
     }
 
+    private fun plusLine(
```

Comment:
So, this thing returns the number of the next line without warnings after `lineNum`?

Reply:
Yes. This method checks if there is another warning after the warning or if there is a line with the code and returns its number.